Dockerfile

```diff
@@ -14,11 +14,11 @@ RUN apt-get update && apt-get upgrade -y
 
 EXPOSE 5000
 
-FROM basepy as prodpy
-ENTRYPOINT poetry run gunicorn "todo_app.app:create_app()" -b 0.0.0.0:5000
-
 FROM basepy as devpy
 ENTRYPOINT poetry run flask run --host 0.0.0.0
 
 FROM basepy as devtestpy
-ENTRYPOINT poetry run pytest
\ No newline at end of file
+ENTRYPOINT poetry run pytest
+
+FROM basepy as prodpy
```

Comment:
Switching the order resolves the issue with the heroku-deploy action not having a solution for specifying a target, but it's not obvious that this matters and future developers might add a new stage and be surprised they break the prod build - adding a quick comment to make that clear would be useful here